bubble_sort/bubble_sort.py

```diff
@@ -0,0 +1,25 @@
+# Python program for implementation of Bubble Sort
+ 
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+ 
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+ 
+            # traverse the array from 0 to n-i-1
+            # Swap if the element found is greater
+            # than the next element
+            if arr[j] > arr[j+1] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+ 
+# Driver code to test above
+arr = [64, 34, 25, 12, 22, 11, 90]
+ 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
+++ b/tmp/tmpwl1l6gsj/bubble_sort/bubble_sort.py
@@ -17,7 +17,7 @@
  
 # Driver code to test above
 arr = [64, 34, 25, 12, 22, 11, 90]
- 
+
 bubbleSort(arr)
  
 print ("Sorted array is:")
```